options.go

```diff
@@ -177,6 +177,28 @@ func WithStartupCommand(execs ...Executable) CustomizeRequestOption {
 	}
 }
 
+// WithAfterReadyCommand will execute the command representation of each Executable into the container.
```

Comment:
I prefer `WithAfterReadyCommand`
```suggestion
// WithReadyCommand will execute the command representation of each Executable into the container.
```

Reply:
Changed. I initially had doubts and opted for the shorter version. If it's not clear enough, let's be explicit